packages/itwinui-variables/src/themes/dark-hc.scss

```diff
@@ -4,66 +4,159 @@
 @use '../utils/hex' as *;
 @use '../utils/get-hsl' as *;
 
-$base-colors: (
-  // Background colors - default
-  background-1: hsl(200 12% 20%),
-  background-2: hsl(200 12% 15%),
-  background-3: hsl(200 12% 10%),
-  background-4: hsl(200 12% 25%),
-  background-5: hsl(200 12% 30%),
-  background-1-hover: hsl(0 0% 95%),
-  //
-  // Background colors - statuses subtle
-  background-subtle-primary: hsl(204 14% 44%),
-  background-subtle-positive: hsl(192 9% 43%),
-  background-subtle-warning: hsl(201 6% 44%),
-  background-subtle-negative: hsl(216 7% 44%),
-  //
-  // Foreground colors - default
-  foreground-1: hsl(0 0% 100%),
-  foreground-2: hsl(0 0% 95%),
-  foreground-3: hsl(210 4% 90%),
-  foreground-4: hsl(200 3% 81%),
-  foreground-5: hsl(195 3% 47%),
-  //
-  // Foreground colors - statuses
-  foreground-primary: hsl(200 100% 79%),
-  foreground-positive: hsl(140 100% 72%),
-  foreground-warning: hsl(45 100% 75%),
-  foreground-negative: hsl(0 100% 88%),
-  foreground-primary-hover: hsl(203 100% 34%),
-  foreground-positive-hover: hsl(95 72% 27%),
-  foreground-warning-hover: hsl(33 89% 41%),
-  foreground-negative-hover: hsl(0 91% 33%)
-);
+// #region SASS variables
+$bg-hue: 203;
+$bg-saturation: 6%; // Less saturated than base theme.
+$bg-lightness-gap: 4.25%;
+$fg-lightness-gap: 5%;
+// #endregion
+
+// #region Background colors
+// Darkest @ top, lightest @ bottom
+$bg-0: hsl($bg-hue $bg-saturation 0%);
+$bg-1: hsl($bg-hue $bg-saturation ($bg-lightness-gap));
+$bg-2: hsl($bg-hue $bg-saturation ($bg-lightness-gap * 2));
+$bg-3: hsl($bg-hue $bg-saturation ($bg-lightness-gap * 3));
+$bg-4: hsl($bg-hue $bg-saturation ($bg-lightness-gap * 4));
+$bg-5: hsl($bg-hue $bg-saturation ($bg-lightness-gap * 5));
+$bg-6: hsl($bg-hue $bg-saturation ($bg-lightness-gap * 5.5));
```

Comment:
Have you thought about moving this color to be outside of the series? Maybe call this one `$bg-5b` instead of `$bg-6`?